src/auth/computeclient.ts

```diff
@@ -20,6 +20,8 @@ import {RequestError} from './../transporters';
 import {CredentialRequest, Credentials} from './credentials';
 import {GetTokenResponse, OAuth2Client} from './oauth2client';
 
+export interface ComputeOptions { refreshTokenEarlyMillis?: number; }
```

Comment:
It would be good to add documentation on what the purpose of this configuration option is and how it behaves.

nit: it would be good if this type was not duplicated in three places. Can you see if you can declare a single interface with the common option (just this one) and make others inherit from it.

Reply:
Done.